download/index.html

```diff
@@ -0,0 +1,130 @@
+    </span>
+  </div>
+
+  <div class="ui basic segment">
+    <div class="row">
+      <div class="ui two column very relaxed grid">
+        <div class="column">
+          <div class="ui center aligned grid">
+            <div class="row">
+              <div class="sixteen wide column">
+                <h4>Earlier Windows</h4>
+                <span class="ui small text">
+                  <i class="info circle icon"></i> Core functional and supported.<br>
+                  Notifications and Prompts are not implemented. For the best experience, do not set the default action to "ask" until a workaround is available.
+                </span>
+              </div>
+            </div>
+            <div class="two column doubling row">
+              <div class="column">
+                <a href="https://updates.safing.io/latest/windows_amd64/packages/portmaster-installer.exe" target="_blank" class="ui button">
+                  <i class="windows icon"></i>
+                  Windows&nbsp;7
+                </a>
+              </div>
+              <div class="column">
+                <a href="https://updates.safing.io/latest/windows_amd64/packages/portmaster-installer.exe" target="_blank" class="ui button">
+                  <i class="windows icon"></i>
+                  Windows&nbsp;8
+                </a>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="column">
+          <div class="ui center aligned grid">
+            <div class="row">
+              <div class="sixteen wide column">
+                <h4>Linux 2.4+</h4>
+                <span class="ui small text">
+                  <i class="info circle icon"></i> Core functional and supported.<br>
+                  Desktop environment integration varies heavily. Currently only packaged for Debian and its derivatives.
+                </span>
+              </div>
+            </div>
+            <div class="three column stackable row">
+              <div class="column">
+                <a href="https://updates.safing.io/latest/linux_amd64/packages/portmaster-installer.deb" target="_blank" class="ui button">
+                  XFCE
+                </a>
+              </div>
+              <div class="column">
+                <a href="https://updates.safing.io/latest/linux_amd64/packages/portmaster-installer.deb" target="_blank" class="ui button">
+                  KDE
+                </a>
+              </div>
+              <div class="column">
+                <a href="https://updates.safing.io/latest/linux_amd64/packages/portmaster-installer.deb" target="_blank" class="ui button">
+                  Gnome
+                </a>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
```

Comment:
I think I might not have communicated correctly what is planned for this section.

We don't need a separate download button for every Desktop Environment, rather, we need a separate description. While the text is currently the same for all three, this will change quickly, as every DE will need different packages for compatibility, or different things might be broken. These will also updated frequently.

Actually, you can already update the XFCE description to: `Notifications have been seen working.`

On the other Hand, it would be great to have one (atm) download button under `Linux 2.4+` called `.deb Installer`, because other installer types will join it.

Reply:
Also, it would make sense to style the Windows segment similarly, as we will probably have different things to say there too.

Reply:
we discussed this in a video call and came to a solution involving linking to a Windows Wiki / Linux Wiki - so the link stays static and the wiki can quickly adapt in the background. So less commits, less PR, more up-to-date info

Reply:
<img width="1144" alt="limited_support" src="https://user-images.githubusercontent.com/3080765/78643462-d2365580-78a3-11ea-80c5-4142e00bc5eb.png">

this is the current version of the **Limited Support** Section. Do you have any feedback, especially in regards to the wording of the lists.